tech/languages/csharp/csharp.md

```diff
@@ -0,0 +1,51 @@
+---
+title: C#
```

Comment:
Please see [File structure](http://developer-portal.github.io/doc/file-structure.html) in our guide.